velox/exec/tests/WindowFunctionRegistryTest.cpp

```diff
@@ -17,7 +17,8 @@
 
 #include "velox/exec/WindowFunction.h"
 #include "velox/expression/SignatureBinder.h"
-#include "velox/type/Type.h"
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
```

Comment:
Is this include file necessary ?

Reply:
Thanks for pointing it out, the include file is not needed and is removed now.